src/main.ts

```diff
@@ -36,52 +36,47 @@ Vue.use(Affix)
 Vue.use(Vuelidate)
 Vue.use(Vue2Filters)
 
-//
-// Fetch config from server, then load Vue.
-//
-fetchConfig()
-  .then(async () => {
-    // initialize Sentry
-    console.info('Initializaing Sentry...') // eslint-disable-line no-console
-    Sentry.init({
-      dsn: window['sentryDsn'],
-      integrations: [new Integrations.Vue({ Vue, attachProps: true })]
-    })
-    // initialize Launch Darkly
-    await initLDClient()
+// main code
+async function start () {
+  // fetch config from environment and API
+  // must come first as inits below depend on config
+  await fetchConfig()
 
-    if (featureFlags.getFlag('bcrs-create-ui-enabled')) {
-      // configure KeyCloak Service
-      console.info('Starting Keycloak service...') // eslint-disable-line no-console
-      await KeycloakService.setKeycloakConfigUrl(sessionStorage.getItem('KEYCLOAK_CONFIG_PATH'))
+  // initialize Sentry
+  console.info('Initializing Sentry...') // eslint-disable-line no-console
+  Sentry.init({
+    dsn: window['sentryDsn'],
+    integrations: [new Integrations.Vue({ Vue, attachProps: true })]
+  })
 
-      // get Vue objects only after we have config
-      const router = getVueRouter()
-      const store = getVuexStore()
+  // initialize Launch Darkly
+  await initLDClient()
 
-      // start Vue application
-      console.info('Starting app...') // eslint-disable-line no-console
-      new Vue({
-        vuetify: new Vuetify({ iconfont: 'mdi' }),
-        router,
-        store,
-        render: h => h(App)
-      }).$mount('#app')
-    }
-  })
-  .catch(error => {
```

Comment:
I think this previous code didn't catch all possible errors. The new code definitely works and is cleaner.